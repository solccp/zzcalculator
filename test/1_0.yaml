---
structure_id: 1
number_of_atoms: 24
double_bonds:
- [1, 20]
- [5, 11]
- [18, 14]
- [22, 21]
- [3, 9]
- [7, 6]
- [13, 17]
- [10, 4]
- [2, 8]
- [12, 19]
- [15, 23]
- [16, 24]
---
structure_id: 2
number_of_atoms: 24
double_bonds:
- [1, 20]
- [5, 11]
- [18, 14]
- [22, 21]
- [3, 7]
- [6, 13]
- [17, 10]
- [2, 8]
- [4, 12]
- [9, 16]
- [19, 15]
- [23, 24]
---
structure_id: 3
number_of_atoms: 24
double_bonds:
- [1, 20]
- [5, 11]
- [18, 14]
- [22, 21]
- [3, 7]
- [6, 13]
- [17, 10]
- [2, 4]
- [12, 19]
- [8, 15]
- [9, 16]
- [23, 24]
---
structure_id: 4
number_of_atoms: 24
double_bonds:
- [1, 20]
- [5, 11]
- [18, 14]
- [22, 21]
- [3, 7]
- [6, 13]
- [17, 10]
- [2, 4]
- [12, 19]
- [8, 9]
- [15, 23]
- [16, 24]
---
structure_id: 5
number_of_atoms: 24
double_bonds:
- [1, 20]
- [5, 11]
- [18, 14]
- [22, 21]
- [3, 7]
- [6, 13]
- [17, 10]
- [2, 4]
- [12, 19]
rings:
- [8, 9, 16, 24, 23, 15]
---
structure_id: 6
number_of_atoms: 24
double_bonds:
- [1, 20]
- [5, 11]
- [18, 14]
- [22, 21]
- [3, 7]
- [6, 13]
- [17, 10]
- [9, 16]
- [23, 24]
rings:
- [2, 4, 12, 19, 15, 8]
---
structure_id: 7
number_of_atoms: 24
double_bonds:
- [1, 20]
- [5, 11]
- [18, 14]
- [22, 21]
- [3, 7]
- [2, 6]
- [13, 17]
- [10, 4]
- [12, 19]
- [8, 15]
- [9, 16]
- [23, 24]
---
structure_id: 8
number_of_atoms: 24
double_bonds:
- [1, 20]
- [5, 11]
- [18, 14]
- [22, 21]
- [3, 7]
- [2, 6]
- [13, 17]
- [10, 4]
- [12, 19]
- [8, 9]
- [15, 23]
- [16, 24]
---
structure_id: 9
number_of_atoms: 24
double_bonds:
- [1, 20]
- [5, 11]
- [18, 14]
- [22, 21]
- [3, 7]
- [2, 6]
- [13, 17]
- [10, 4]
- [12, 19]
rings:
- [8, 9, 16, 24, 23, 15]
---
structure_id: 10
number_of_atoms: 24
double_bonds:
- [1, 20]
- [5, 11]
- [18, 14]
- [22, 21]
- [3, 7]
- [12, 19]
- [8, 15]
- [9, 16]
- [23, 24]
rings:
- [2, 6, 13, 17, 10, 4]
---
structure_id: 11
number_of_atoms: 24
double_bonds:
- [1, 20]
- [5, 11]
- [18, 14]
- [22, 21]
- [3, 7]
- [12, 19]
- [8, 9]
- [15, 23]
- [16, 24]
rings:
- [2, 6, 13, 17, 10, 4]
---
structure_id: 12
number_of_atoms: 24
double_bonds:
- [1, 20]
- [5, 11]
- [18, 14]
- [22, 21]
- [3, 7]
- [12, 19]
rings:
- [2, 6, 13, 17, 10, 4]
- [8, 9, 16, 24, 23, 15]
---
structure_id: 13
number_of_atoms: 24
double_bonds:
- [1, 20]
- [5, 11]
- [18, 14]
- [22, 21]
- [13, 17]
- [10, 4]
- [12, 19]
- [15, 23]
- [16, 24]
rings:
- [3, 7, 6, 2, 8, 9]
---
structure_id: 14
number_of_atoms: 24
double_bonds:
- [1, 20]
- [3, 5]
- [11, 18]
- [2, 8]
- [9, 16]
- [24, 23]
- [15, 19]
- [12, 4]
- [10, 17]
- [6, 13]
- [7, 14]
- [21, 22]
---
structure_id: 15
number_of_atoms: 24
double_bonds:
- [1, 20]
- [3, 5]
- [11, 18]
- [2, 8]
- [9, 16]
- [24, 23]
- [15, 19]
- [12, 4]
- [10, 17]
- [6, 7]
- [13, 21]
- [14, 22]
---
structure_id: 16
number_of_atoms: 24
double_bonds:
- [1, 20]
- [3, 5]
- [11, 18]
- [2, 8]
- [9, 16]
- [24, 23]
- [15, 19]
- [12, 4]
- [10, 17]
rings:
- [6, 7, 14, 22, 21, 13]
---
structure_id: 17
number_of_atoms: 24
double_bonds:
- [1, 20]
- [3, 5]
- [11, 18]
- [2, 4]
- [10, 17]
- [12, 19]
- [6, 13]
- [7, 14]
- [21, 22]
- [8, 15]
- [9, 16]
- [23, 24]
---
structure_id: 18
number_of_atoms: 24
double_bonds:
- [1, 20]
- [3, 5]
- [11, 18]
- [2, 4]
- [10, 17]
- [12, 19]
- [6, 13]
- [7, 14]
- [21, 22]
- [8, 9]
- [15, 23]
- [16, 24]
---
structure_id: 19
number_of_atoms: 24
double_bonds:
- [1, 20]
- [3, 5]
- [11, 18]
- [2, 4]
- [10, 17]
- [12, 19]
- [6, 13]
- [7, 14]
- [21, 22]
rings:
- [8, 9, 16, 24, 23, 15]
---
structure_id: 20
number_of_atoms: 24
double_bonds:
- [1, 20]
- [3, 5]
- [11, 18]
- [2, 4]
- [10, 17]
- [12, 19]
- [6, 7]
- [13, 21]
- [14, 22]
- [8, 15]
- [9, 16]
- [23, 24]
---
structure_id: 21
number_of_atoms: 24
double_bonds:
- [1, 20]
- [3, 5]
- [11, 18]
- [2, 4]
- [10, 17]
- [12, 19]
- [6, 7]
- [13, 21]
- [14, 22]
- [8, 9]
- [15, 23]
- [16, 24]
---
structure_id: 22
number_of_atoms: 24
double_bonds:
- [1, 20]
- [3, 5]
- [11, 18]
- [2, 4]
- [10, 17]
- [12, 19]
- [6, 7]
- [13, 21]
- [14, 22]
rings:
- [8, 9, 16, 24, 23, 15]
---
structure_id: 23
number_of_atoms: 24
double_bonds:
- [1, 20]
- [3, 5]
- [11, 18]
- [2, 4]
- [10, 17]
- [12, 19]
- [8, 15]
- [9, 16]
- [23, 24]
rings:
- [6, 7, 14, 22, 21, 13]
---
structure_id: 24
number_of_atoms: 24
double_bonds:
- [1, 20]
- [3, 5]
- [11, 18]
- [2, 4]
- [10, 17]
- [12, 19]
- [8, 15]
- [9, 16]
- [23, 24]
- [8, 9]
- [15, 23]
- [16, 24]
---
structure_id: 25
number_of_atoms: 24
double_bonds:
- [1, 20]
- [3, 5]
- [11, 18]
- [2, 4]
- [10, 17]
- [12, 19]
- [8, 15]
- [9, 16]
- [23, 24]
rings:
- [8, 9, 16, 24, 23, 15]
---
structure_id: 26
number_of_atoms: 24
double_bonds:
- [1, 20]
- [3, 5]
- [11, 18]
- [9, 16]
- [10, 17]
- [23, 24]
- [6, 13]
- [7, 14]
- [21, 22]
rings:
- [2, 4, 12, 19, 15, 8]
---
structure_id: 27
number_of_atoms: 24
double_bonds:
- [1, 20]
- [3, 5]
- [11, 18]
- [9, 16]
- [10, 17]
- [23, 24]
- [6, 7]
- [13, 21]
- [14, 22]
rings:
- [2, 4, 12, 19, 15, 8]
---
structure_id: 28
number_of_atoms: 24
double_bonds:
- [1, 20]
- [3, 5]
- [11, 18]
- [9, 16]
- [10, 17]
- [23, 24]
rings:
- [2, 4, 12, 19, 15, 8]
- [6, 7, 14, 22, 21, 13]
---
structure_id: 29
number_of_atoms: 24
double_bonds:
- [1, 20]
- [3, 5]
- [11, 18]
- [2, 6]
- [7, 14]
- [22, 21]
- [13, 17]
- [10, 4]
- [12, 19]
- [8, 15]
- [9, 16]
- [23, 24]
---
structure_id: 30
number_of_atoms: 24
double_bonds:
- [1, 20]
- [3, 5]
- [11, 18]
- [2, 6]
- [7, 14]
- [22, 21]
- [13, 17]
- [10, 4]
- [12, 19]
- [8, 9]
- [15, 23]
- [16, 24]
---
structure_id: 31
number_of_atoms: 24
double_bonds:
- [1, 20]
- [3, 5]
- [11, 18]
- [2, 6]
- [7, 14]
- [22, 21]
- [13, 17]
- [10, 4]
- [12, 19]
rings:
- [8, 9, 16, 24, 23, 15]
---
structure_id: 32
number_of_atoms: 24
double_bonds:
- [1, 20]
- [3, 5]
- [11, 18]
- [7, 14]
- [12, 19]
- [21, 22]
- [8, 15]
- [9, 16]
- [23, 24]
rings:
- [2, 6, 13, 17, 10, 4]
---
structure_id: 33
number_of_atoms: 24
double_bonds:
- [1, 20]
- [3, 5]
- [11, 18]
- [7, 14]
- [12, 19]
- [21, 22]
- [8, 9]
- [15, 23]
- [16, 24]
rings:
- [2, 6, 13, 17, 10, 4]
---
structure_id: 34
number_of_atoms: 24
double_bonds:
- [1, 20]
- [3, 5]
- [11, 18]
- [7, 14]
- [12, 19]
- [21, 22]
rings:
- [2, 6, 13, 17, 10, 4]
- [8, 9, 16, 24, 23, 15]
---
structure_id: 35
number_of_atoms: 24
double_bonds:
- [1, 20]
- [22, 21]
- [6, 13]
- [17, 10]
- [2, 8]
- [4, 12]
- [9, 16]
- [19, 15]
- [23, 24]
rings:
- [3, 5, 11, 18, 14, 7]
---
structure_id: 36
number_of_atoms: 24
double_bonds:
- [1, 20]
- [22, 21]
- [6, 13]
- [17, 10]
- [2, 4]
- [12, 19]
- [8, 15]
- [9, 16]
- [23, 24]
rings:
- [3, 5, 11, 18, 14, 7]
---
structure_id: 37
number_of_atoms: 24
double_bonds:
- [1, 20]
- [22, 21]
- [6, 13]
- [17, 10]
- [2, 4]
- [12, 19]
- [8, 9]
- [15, 23]
- [16, 24]
rings:
- [3, 5, 11, 18, 14, 7]
---
structure_id: 38
number_of_atoms: 24
double_bonds:
- [1, 20]
- [22, 21]
- [6, 13]
- [17, 10]
- [2, 4]
- [12, 19]
rings:
- [3, 5, 11, 18, 14, 7]
- [8, 9, 16, 24, 23, 15]
---
structure_id: 39
number_of_atoms: 24
double_bonds:
- [1, 20]
- [22, 21]
- [6, 13]
- [17, 10]
- [9, 16]
- [23, 24]
rings:
- [3, 5, 11, 18, 14, 7]
- [2, 4, 12, 19, 15, 8]
---
structure_id: 40
number_of_atoms: 24
double_bonds:
- [1, 20]
- [22, 21]
- [2, 6]
- [13, 17]
- [10, 4]
- [12, 19]
- [8, 15]
- [9, 16]
- [23, 24]
rings:
- [3, 5, 11, 18, 14, 7]
---
structure_id: 41
number_of_atoms: 24
double_bonds:
- [1, 20]
- [22, 21]
- [2, 6]
- [13, 17]
- [10, 4]
- [12, 19]
- [8, 9]
- [15, 23]
- [16, 24]
rings:
- [3, 5, 11, 18, 14, 7]
---
structure_id: 42
number_of_atoms: 24
double_bonds:
- [1, 20]
- [22, 21]
- [2, 6]
- [13, 17]
- [10, 4]
- [12, 19]
rings:
- [3, 5, 11, 18, 14, 7]
- [8, 9, 16, 24, 23, 15]
---
structure_id: 43
number_of_atoms: 24
double_bonds:
- [1, 20]
- [22, 21]
- [12, 19]
- [8, 15]
- [9, 16]
- [23, 24]
rings:
- [3, 5, 11, 18, 14, 7]
- [2, 6, 13, 17, 10, 4]
---
structure_id: 44
number_of_atoms: 24
double_bonds:
- [1, 20]
- [22, 21]
- [12, 19]
- [8, 9]
- [15, 23]
- [16, 24]
rings:
- [3, 5, 11, 18, 14, 7]
- [2, 6, 13, 17, 10, 4]
---
structure_id: 45
number_of_atoms: 24
double_bonds:
- [1, 20]
- [22, 21]
- [12, 19]
rings:
- [3, 5, 11, 18, 14, 7]
- [2, 6, 13, 17, 10, 4]
- [8, 9, 16, 24, 23, 15]
---
structure_id: 46
number_of_atoms: 24
double_bonds:
- [1, 5]
- [11, 18]
- [20, 16]
- [24, 23]
- [3, 9]
- [2, 8]
- [15, 19]
- [12, 4]
- [10, 17]
- [6, 13]
- [7, 14]
- [21, 22]
---
structure_id: 47
number_of_atoms: 24
double_bonds:
- [1, 5]
- [11, 18]
- [20, 16]
- [24, 23]
- [3, 9]
- [2, 8]
- [15, 19]
- [12, 4]
- [10, 17]
- [6, 7]
- [13, 21]
- [14, 22]
---
structure_id: 48
number_of_atoms: 24
double_bonds:
- [1, 5]
- [11, 18]
- [20, 16]
- [24, 23]
- [3, 9]
- [2, 8]
- [15, 19]
- [12, 4]
- [10, 17]
rings:
- [6, 7, 14, 22, 21, 13]
---
structure_id: 49
number_of_atoms: 24
double_bonds:
- [1, 5]
- [11, 18]
- [20, 16]
- [24, 23]
- [3, 9]
- [2, 4]
- [8, 15]
- [10, 17]
- [12, 19]
- [6, 13]
- [7, 14]
- [21, 22]
---
structure_id: 50
number_of_atoms: 24
double_bonds:
- [1, 5]
- [11, 18]
- [20, 16]
- [24, 23]
- [3, 9]
- [2, 4]
- [8, 15]
- [10, 17]
- [12, 19]
- [6, 7]
- [13, 21]
- [14, 22]
---
structure_id: 51
number_of_atoms: 24
double_bonds:
- [1, 5]
- [11, 18]
- [20, 16]
- [24, 23]
- [3, 9]
- [2, 4]
- [8, 15]
- [10, 17]
- [12, 19]
rings:
- [6, 7, 14, 22, 21, 13]
---
structure_id: 52
number_of_atoms: 24
double_bonds:
- [1, 5]
- [11, 18]
- [20, 16]
- [24, 23]
- [3, 9]
- [10, 17]
- [6, 13]
- [7, 14]
- [21, 22]
rings:
- [2, 4, 12, 19, 15, 8]
---
structure_id: 53
number_of_atoms: 24
double_bonds:
- [1, 5]
- [11, 18]
- [20, 16]
- [24, 23]
- [3, 9]
- [10, 17]
- [6, 7]
- [13, 21]
- [14, 22]
rings:
- [2, 4, 12, 19, 15, 8]
---
structure_id: 54
number_of_atoms: 24
double_bonds:
- [1, 5]
- [11, 18]
- [20, 16]
- [24, 23]
- [3, 9]
- [10, 17]
rings:
- [2, 4, 12, 19, 15, 8]
- [6, 7, 14, 22, 21, 13]
---
structure_id: 55
number_of_atoms: 24
double_bonds:
- [1, 5]
- [11, 18]
- [20, 16]
- [24, 23]
- [3, 9]
- [2, 6]
- [7, 14]
- [8, 15]
- [19, 12]
- [4, 10]
- [17, 13]
- [21, 22]
---
structure_id: 56
number_of_atoms: 24
double_bonds:
- [1, 5]
- [11, 18]
- [20, 16]
- [24, 23]
- [3, 9]
- [7, 14]
- [8, 15]
- [12, 19]
- [21, 22]
rings:
- [2, 6, 13, 17, 10, 4]
---
structure_id: 57
number_of_atoms: 24
double_bonds:
- [1, 5]
- [11, 18]
- [20, 16]
- [24, 23]
- [3, 7]
- [9, 8]
- [14, 22]
- [15, 19]
- [12, 4]
- [2, 6]
- [10, 17]
- [13, 21]
---
structure_id: 58
number_of_atoms: 24
double_bonds:
- [1, 5]
- [11, 18]
- [20, 16]
- [24, 23]
- [14, 22]
- [15, 19]
- [12, 4]
- [10, 17]
- [13, 21]
rings:
- [3, 7, 6, 2, 8, 9]
---
structure_id: 59
number_of_atoms: 24
double_bonds:
- [11, 18]
- [24, 23]
- [2, 8]
- [15, 19]
- [12, 4]
- [10, 17]
- [6, 13]
- [7, 14]
- [21, 22]
rings:
- [1, 5, 3, 9, 16, 20]
---
structure_id: 60
number_of_atoms: 24
double_bonds:
- [11, 18]
- [24, 23]
- [2, 8]
- [15, 19]
- [12, 4]
- [10, 17]
- [6, 7]
- [13, 21]
- [14, 22]
rings:
- [1, 5, 3, 9, 16, 20]
---
structure_id: 61
number_of_atoms: 24
double_bonds:
- [11, 18]
- [24, 23]
- [2, 8]
- [15, 19]
- [12, 4]
- [10, 17]
rings:
- [1, 5, 3, 9, 16, 20]
- [6, 7, 14, 22, 21, 13]
---
structure_id: 62
number_of_atoms: 24
double_bonds:
- [11, 18]
- [24, 23]
- [2, 4]
- [8, 15]
- [10, 17]
- [12, 19]
- [6, 13]
- [7, 14]
- [21, 22]
rings:
- [1, 5, 3, 9, 16, 20]
---
structure_id: 63
number_of_atoms: 24
double_bonds:
- [11, 18]
- [24, 23]
- [2, 4]
- [8, 15]
- [10, 17]
- [12, 19]
- [6, 7]
- [13, 21]
- [14, 22]
rings:
- [1, 5, 3, 9, 16, 20]
---
structure_id: 64
number_of_atoms: 24
double_bonds:
- [11, 18]
- [24, 23]
- [2, 4]
- [8, 15]
- [10, 17]
- [12, 19]
rings:
- [1, 5, 3, 9, 16, 20]
- [6, 7, 14, 22, 21, 13]
---
structure_id: 65
number_of_atoms: 24
double_bonds:
- [11, 18]
- [24, 23]
- [10, 17]
- [6, 13]
- [7, 14]
- [21, 22]
rings:
- [1, 5, 3, 9, 16, 20]
- [2, 4, 12, 19, 15, 8]
---
structure_id: 66
number_of_atoms: 24
double_bonds:
- [11, 18]
- [24, 23]
- [10, 17]
- [6, 7]
- [13, 21]
- [14, 22]
rings:
- [1, 5, 3, 9, 16, 20]
- [2, 4, 12, 19, 15, 8]
---
structure_id: 67
number_of_atoms: 24
double_bonds:
- [11, 18]
- [24, 23]
- [10, 17]
rings:
- [1, 5, 3, 9, 16, 20]
- [2, 4, 12, 19, 15, 8]
- [6, 7, 14, 22, 21, 13]
---
structure_id: 68
number_of_atoms: 24
double_bonds:
- [11, 18]
- [24, 23]
- [2, 6]
- [7, 14]
- [8, 15]
- [19, 12]
- [4, 10]
- [17, 13]
- [21, 22]
rings:
- [1, 5, 3, 9, 16, 20]
---
structure_id: 69
number_of_atoms: 24
double_bonds:
- [11, 18]
- [24, 23]
- [7, 14]
- [8, 15]
- [12, 19]
- [21, 22]
rings:
- [1, 5, 3, 9, 16, 20]
- [2, 6, 13, 17, 10, 4]
