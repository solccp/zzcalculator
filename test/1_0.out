 20 + 32 x + 15 x^2 + 2 x^3
 total: 69
